UIforETW/Utility.cpp

```diff
@@ -547,9 +770,9 @@ std::wstring GetBuildTimeFromAddress(void* codeAddress)
 	// Print out the module information. The %.24s is necessary to trim
 	// the new line character off of the date string returned by asctime().
 	// _wasctime_s requires a 26-character buffer.
-	wchar_t ascTimeBuf[26];
+	wchar_t ascTimeBuf[26] = {0};

```

Comment:
I try to avoid "= {0}" - prefer "= {}". But, not a big deal.

I'm also curious as to why you initialize these at all? The _s functions guarantee null-termination, always, so this is provably unneeded waste.